### PR TITLE
1835 updates on entities, first SR

### PR DESCRIPTION
**Pull request by volker18xxdev**
first updates for 1835. entities, first SR,

### CODE REVIEW

File: lib/engine/game/g_1835/entities.rb [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+      module Entities
+        COMPANIES = [
+            {
+              name: 'Nürnberg-Fürth',
+              sym: 'NF',
+              value: 100,
+              revenue: 5,
+              desc: 'Nürnberg-Fürth Bahn, Director of AG may lay token on L14 north or south, 10% Bayern',
+              abilities: [{ type: 'shares', shares: 'BY_2' }, { type: 'no_buy' }],
+              color: nil,
```
Comment (jenf) [inline lib/engine/game/g_1835/entities.rb:15]: Set color, otherwise the UI will go a bit wonky.

File: lib/engine/game/g_1835/entities.rb [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+            name: 'Bayrische Eisenbahn',
+            logo: '1835/BY',
+            simple_logo: '1835/BY.alt',
+            max_ownership_percent: 100,
+            tokens: [0, 0, 0, 0, 0],
+            float_percent: 50,
+            shares: [20, 10, 10, 10, 10, 10, 10, 10, 10],
+            coordinates: 'O15',
+            color: :Blue,
+            reservation_color: nil,
```
Comment (jenf) [inline lib/engine/game/g_1835/entities.rb:205]: remove nils

File: lib/engine/game/g_1835/entities.rb [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+            sym: 'HE',
+            name: 'Hessische Eisenbahn',
+            logo: '1835/HE',
+            simple_logo: '1835/HE.alt',
+            tokens: [0, 0],
+            max_ownership_percent: 100,
+            float_percent: 50,
+            shares: [20, 10, 10, 10, 10, 10, 10, 20],
+            last_cert: %w[HE_7],
+            # last_cert = true,
```
Comment (jenf) [inline lib/engine/game/g_1835/entities.rb:244]: either remove or change to a todo

File: lib/engine/game/g_1835/round/operating.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require_relative '../../../round/operating'
+
+module Engine
+  module Game
+    module G1835
```
Comment (jenf) [inline lib/engine/game/g_1835/round/operating.rb:7]: Don't override if unnecessary to do so

Comment (volker18xxdev) [reply lib/engine/game/g_1835/round/operating.rb:7]: without this file, there is not operating round possible, only stockrounds.

File: lib/engine/game/g_1835/entities.rb (hunk 3 above)
Comment (volker18xxdev) [reply lib/engine/game/g_1835/entities.rb:244]: removed

File: lib/engine/game/g_1835/entities.rb (hunk 2 above)
Comment (volker18xxdev) [reply lib/engine/game/g_1835/entities.rb:205]: removed

File: lib/engine/game/g_1835/entities.rb (hunk 1 above)
Comment (volker18xxdev) [reply lib/engine/game/g_1835/entities.rb:15]: removed

File: lib/engine/game/g_1835/round/operating.rb (hunk 4 above)
Comment (jenf) [reply lib/engine/game/g_1835/round/operating.rb:7]: you can just use the generic one, you don't need to subclass if it's unaltered.

Comment (volker18xxdev) [reply lib/engine/game/g_1835/round/operating.rb]: i found the right place.

File: lib/engine/game/g_1835/round/draft.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+require_relative '../../../round/draft'
+
+module Engine
+  module Game
+    module G1835
+      module Round
+        class Draft < Engine::Round::Draft
+          def select_entities
+            @game.players
```
Comment (jenf) [inline lib/engine/game/g_1835/round/draft.rb:11]: This looks the same as the base class (assuming reverse is set to false)

Could you remove?

**Review by jenf — CHANGES_REQUESTED** [review]
One minor change, otherwise LGTM.

File: lib/engine/game/g_1835/round/draft.rb (hunk 5 above)
Comment (volker18xxdev) [reply lib/engine/game/g_1835/round/draft.rb:11]: there is a variant "Clemens", where reverse is true, not fully implented.

**Review by jenf — APPROVED** [review]

**Pull request merged**